hepcrawl/testlib/tasks.py

```diff
@@ -19,11 +19,8 @@
 
 
 class Config(object):
-    CELERY_RESULT_BACKEND = "amqp://guest:guest@rabbitmq:5672//"
-    BROKER_URL = "amqp://guest:guest@rabbitmq:5672//"
-    CELERY_ALWAYS_EAGER = True
-    CELERY_CACHE_BACKEND = 'memory'
-    CELERY_EAGER_PROPAGATES_EXCEPTIONS = True
```

Comment:
These three configuration variables have not had effect for a long time, as they are not valid or ignored in Celery 4, so we can safely remove them.